saidump/saidump.cpp

```diff
@@ -82,6 +95,11 @@ CmdOptions handleCmdLine(int argc, char **argv)
                 options.dumpTempView = true;
                 break;
 
+            case 'r':
+                SWSS_LOG_NOTICE("Dumping from dump.rdb");
```

Comment:
if the file is a parameter, you should use actual path not dump.rdb hardcoded

Reply:
Fix it.